scripts/ai/controllers/CombineController.lua

```diff
@@ -156,14 +178,15 @@ function CombineController:fixDischargeDistanceForChopper()
     local dischargeNode = self.implement:getCurrentDischargeNode()
     if self:isChopper() and dischargeNode and dischargeNode.maxDistance then
         local safeDischargeNodeMaxDistance = 40
-        if dischargeNode.maxDistance < safeDischargeNodeMaxDistance then
+        if false and dischargeNode.maxDistance < safeDischargeNodeMaxDistance then
             self:debug('Chopper maximum throw distance is %.1f, increasing to %.1f', dischargeNode.maxDistance, safeDischargeNodeMaxDistance)
             dischargeNode.maxDistance = safeDischargeNodeMaxDistance
         end
     end
 end
 
 function CombineController:isChopper()
```

Comment:
Do we want to remove this then ? Or is there any use in trying to separate a choper with the dewulf and so on?

Reply:
Removed call but left the function in, we'll see later if we need it.